package-res/resources/web/pentaho/type/Item.js

```diff
@@ -57,7 +57,7 @@ define([
    * <caption> Create a new class <code>DerivedItem</code> containing
    * an attribute <code>greeting</code> and a method <code>doSomething</code> </caption>
    *
-   * require(["pentaho/type/Item"], function(Item) {
+   * require(["pentaho/type/Item", "pentaho/util/logger"], function(Item, logger) {
```

Comment:
This is just an example. I don't think logger is public is it? Even if it were, it's just an example.